src/model/VersionNumber.ts

```diff
@@ -1,6 +1,4 @@
-import ReactiveObjectConverterInterface from './safety/ReactiveObjectConverter';
-
-export default class VersionNumber implements ReactiveObjectConverterInterface {
+export default class VersionNumber {
```

Comment:
Didn't the earlier version of the PR also remove `fromReactive` from here? Why was that change dropped, I think the method should still go from this class.

Reply:
Whoops, I did an oopsie when reverting changes